public/style.css

```diff
@@ -0,0 +1,34 @@
+body{
+  background-image: url('https://media.giphy.com/media/Kto2WhhTPwRcQ/giphy.gif');
+  background-size: cover;
+  height: 100vh;
+  padding:0;
+  margin:0;
+}
+h1, h2{
+font-family: 'Smokum', cursive;
+text-align: center;
+color: aliceblue;
+font-size: 100px;
+}
+
+button, input{
+  font-family: 'Pacifico', cursive;
+  font-size: 20px;
+
+}
+
+h1,h2{
+  margin-top: 0px; 
```

Comment:
Just a css shortcut when you need to put margins 0 for top/bottom/left/right you can use `margin:0;`